test/midcom/helper/imagepopup/handler/uploadTest.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * @package openpsa.test
+ * @author CONTENT CONTROL http://www.contentcontrol-berlin.de/
+ * @copyright CONTENT CONTROL http://www.contentcontrol-berlin.de/
+ * @license http://www.gnu.org/licenses/gpl.html GNU General Public License
+ */
+
+/**
+ * OpenPSA testcase
+ *
+ * @package openpsa.test
+ */
+class midcom_helper_imagepopup_handler_uploadTest extends openpsa_testcase
+{
+    protected static $_images;
+    protected static $_tmp_folder;
+    protected static $_tmp_names;
+    
+    public static function setupBeforeClass()
+    {
+        self::$_images = array();
+        self::$_tmp_names = array();
+        // We need two images to run tests
+        self::$_images = self::create_images(2);
+    }
+    
+    public function testHandler_upload()
+    {
+        midcom::get()->auth->request_sudo('midcom.helper.imagepopup');
+        $node = self::get_component_node('net.nehmer.static');
+        $_FILES = self::$_images;
+        
+        // Test images really exists ?
+        foreach(self::$_tmp_names as $temp_name)
+        {
+            $this->assertTrue(file_exists($temp_name));
+            $this->assertTrue(is_writable($temp_name));
+        }
+        
+        // Do it goes with guid ?
+        $data = $this->run_handler('net.nehmer.static', array ('__ais', 'imagepopup', 'upload', 'image', $node->guid));
+        $this->assertEquals('____ais-imagepopup-upload_image', $data['handler_id']);
+       
+        // Do it goes without guid ?
+        $data = $this->run_handler('net.nehmer.static', array ('__ais', 'imagepopup', 'upload', 'image'));
+        $this->assertEquals('____ais-imagepopup-upload_image_noguid', $data['handler_id']);
+        
```

Comment:
could you also test if the handler really did what it was supposed to do? for example, did it make a new attachment?